sdcm/cluster.py

```diff
@@ -2149,39 +2144,53 @@ def is_scylla_installed(self, raise_if_not_installed=False):
             raise Exception(f"There is no pre-installed ScyllaDB on {self}")
         return False
 
-    def get_scylla_version(self) -> str:
-        self.scylla_version = self.scylla_version_detailed = ""
-        scylla_path = self.add_install_prefix("/usr/bin/scylla")
-        result = self.remoter.run(f"{scylla_path} --version", ignore_status=True)
+    def get_scylla_binary_version(self) -> Optional[str]:
+        result = self.remoter.run(f"{self.add_install_prefix('/usr/bin/scylla')} --version", ignore_status=True)
         if result.ok:
-            self.scylla_version_detailed = result.stdout.strip()
-            if build_id := self.get_scylla_build_id():
-                self.scylla_version = self.scylla_version_detailed
-                self.scylla_version_detailed += f" with build-id {build_id}"
-                self.log.info(f"Found ScyllaDB version with details: {self.scylla_version_detailed}")
-            self.log.debug(f"self.scylla_version_detailed={self.scylla_version_detailed}")
-        else:
-            if self.distro.is_rhel_like:
-                cmd = f"rpm --query --queryformat '%{{VERSION}}' {self.scylla_pkg()}"
-            else:
-                cmd = f"dpkg-query --show --showformat '${{Version}}' {self.scylla_pkg()}"
-            result = self.remoter.run(cmd, ignore_status=True)
+            return result.stdout.strip()
+        self.log.debug("Unable to get ScyllaDB version using `%s':\n%s\n%s",
```

Comment:
Not critical, but is it intended that there is different quotes used in this line for wrapping first substitution place:

```
`%s'
```
?

Reply:
Yes, it's intended. I never thought about the source of this habit because quotations like this was everywhere around in pure ASCII time, but maybe [this](https://unix.stackexchange.com/questions/73989/why-do-unix-man-pages-use-double-backticks-in-place-of-double-quotes) and [this](https://stackoverflow.com/questions/15390066/why-does-the-shell-output-often-use-mixed-characters-in-its-output) will answer your question better.